RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+
+This project produces GitHub Releases as the versioned artifacts for use in products like the Terraform VS Code Extension.
+
+For example, you can download the v0.2.3 Terraform Syntax release by using `
+https://github.com/hashicorp/syntax/releases/download/v0.2.3/terraform.tmGrammar.json`
```

Comment:
```suggestion
For example, you can download the `v0.2.3` Terraform Syntax release from `https://github.com/hashicorp/syntax/releases/download/v0.2.3/terraform.tmGrammar.json`
```
Not sure if you intended to say "by using curl" here or something like that, but I guess the URL may be sufficient - I just fixed the formatting.

Reply:
Did not intend to give example ways yet. We're still working everything out and the RELEASE.md document isn't really going to be followed by external people anyway.